sdk/standbypool/Azure.ResourceManager.StandbyPool/samples/Generated/Samples/Sample_StandbyContainerGroupPoolCollection.cs

```diff
@@ -23,7 +23,7 @@ public partial class Sample_StandbyContainerGroupPoolCollection
         public async Task CreateOrUpdate_StandbyContainerGroupPoolsCreateOrUpdate()
         {
             // Generated from example definition: 2024-03-01/StandbyContainerGroupPools_CreateOrUpdate.json
-            // this example is just showing the usage of "StandbyContainerGroupPoolResource_CreateOrUpdate" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "StandbyContainerGroupPools_createOrUpdate" operation, for the dependent resources, they will have to be created separately.
```

Comment:
it's `createOrUpdate`: https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/standbypool/StandbyPool.Management/standbyContainerGroupPool.tsp#L111